### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by matfij**
- update packages to latest non-breaking change
- udpate eslint to new major version & update eslint config
- remove deprecated sass APIs 

image

___

takeaways:
- a more-restricted color palette should be used to prevent large migration & ensure consistency (new story)
- even large, well-established libraries will change in unexpected manner -> create abstractions for their APIs

___

identified issues:
- `vite` is using older `Dart Sass` version which will be not supported in `sass` 2.0.0+ -> we will have to make sure `vite` is supporting it before updating

### CODE REVIEW

File: eslint.config.js
Comment (domgan) [inline eslint.config.js:1]: Wasn't that changed in a different PR? Where's no floating promises?

**Review by domgan — APPROVED** [review]

File: eslint.config.js
Comment (matfij) [reply eslint.config.js:1]: missed, added (we didn't have it before)

File: src/common/styles/forms.scss [hunk 1]
```diff
@@ -31,7 +32,7 @@ fieldset {
     appearance: none;
     padding: 0.5rem;
     font-size: 16px;
-    border: 1px solid darken($color-light, 25);
+    border: 1px solid color.adjust($color-light, $lightness: -25%);
```
[the remark this answers is not in the document]
Comment (matfij) [reply src/common/styles/forms.scss:35]: we will need to centralize it

File: eslint.config.js
Comment (domgan) [reply eslint.config.js:1]: Ah I see. that's client and it was server before xD

**Pull request merged**